packages/xo-web/src/common/intl/locales/fr.js

```diff
@@ -3312,7 +3312,7 @@ export default {
 
   // Original text: "You can download our turnkey appliance at {website}"
   downloadXoaFromWebsite:
-    'Téléchargez notre édition clef en main sur {website}',
+    'Téléchargez l’appareil clé en main à {website}',
```

Comment:
"Appliance" in IT world isn't translated by "Appareil" in French. So the original sentence is still correct.

Reply:
Changes made